dspace-spring-rest/src/test/java/org/dspace/app/rest/RelationshipTypeRestRepositoryIT.java

```diff
@@ -0,0 +1,254 @@
+
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.isEmptyOrNullString;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.sql.SQLException;
+import java.util.List;
+
+import org.dspace.app.rest.matcher.EntityTypeMatcher;
+import org.dspace.app.rest.matcher.RelationshipTypeMatcher;
+import org.dspace.app.rest.test.AbstractEntityIntegrationTest;
+import org.dspace.content.RelationshipType;
+import org.dspace.content.service.EntityTypeService;
+import org.dspace.content.service.RelationshipTypeService;
+import org.h2.util.StringUtils;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class RelationshipTypeRestRepositoryIT extends AbstractEntityIntegrationTest {
+
+    @Autowired
+    private RelationshipTypeService relationshipTypeService;
+
+    @Autowired
+    private EntityTypeService entityTypeService;
+
+    @Test
+    public void findAllRelationshipTypesTest() throws SQLException {
+        assertEquals(10, relationshipTypeService.findAll(context).size());
+    }
+
+    @Test
+    public void findPublicationPersonRelationshipType() throws SQLException {
+        String leftTypeString = "Publication";
+        String rightTypeString = "Person";
+        String leftLabel = "isAuthorOfPublication";
+        String rightLabel = "isPublicationOfAuthor";
+        checkRelationshipType(leftTypeString, rightTypeString, leftLabel, rightLabel);
```

Comment:
this is an unit test. There is no value in checking all the setup data individually, please keep only one and move to the test to the right location